x11-packages/firefox/build.sh

```diff
@@ -66,7 +61,7 @@ termux_step_configure() {
 		--with-system-webp \
 		--with-system-zlib \
 		--without-wasm-sandboxed-libraries \
-		--with-branding=browser/branding/aurora \
+		--with-branding=browser/branding/official \
```

Comment:
Does not this cause legal issues? (I'm not familiar with the Firefox Branding policy.)

Reply:
I don't know too. But I doubt that this is illegal.
Seeing build scripts of Debian and Arch Linux, they both enable official branding (```--enable-official-branding```).
```Branding``` seems to be explained here https://firefox-source-docs.mozilla.org/browser/branding/docs/index.html.

Reply:
OK then go ahead.